tests/ConfigTest.php

```diff
@@ -107,7 +107,7 @@ public function testThatFinderWorksWithDirSetOnConfig()
         $config = new Config();
 
         $iterator = $config->getFinder()->in(__DIR__.'/Fixtures/FinderDirectory')->getIterator();
-        $this->assertSame(1, count($iterator));
+        $this->assertSame(1, iterator_count($iterator));
```

Comment:
[`assertCount`](https://github.com/sebastianbergmann/phpunit/blob/4.0/src/Framework/Constraint/Count.php#L105) will work I think, however we might run in the old HHVM bug